inc/hooks.php

```diff
@@ -43,7 +43,18 @@ function ppom_hooks_save_cropped_image( $ppom_fields, $posted_data ) {
 
 				$image_data = $values['cropped'];
 				$file_name  = isset( $values['org'] ) ? $values['org'] : '';
-				$file_name  = ppom_file_get_name( $file_name, $product_id );
+
+				// Verify uploaded file type.
+				$allowed_types = isset( $cropper['file_types'] ) ? sanitize_text_field( $cropper['file_types'] ) : 'jpg,png';
```

Comment:
Let's also bring back the `empty` check from the previous iteration, along with `isset`.